nosetests_answers.ini

```diff
@@ -15,4 +15,5 @@ author.irc = irc.freenode.org#plone
 
 plone.version = 4.3.6
 plone.is_plone5 = False
+plone.minor_version = 4.3
```

Comment:
We *are* extracting it from plone.version. But I guess when you use a .ini file, the code that extracts it (`post_plone_version` in `hooks.py`) is not getting called.